src/nre.nim

```diff
@@ -427,7 +416,9 @@ proc initRegex(pattern: string, options: string): Regex =
 
   result.captureNameToId = result.getNameToNumberTable()
 
-proc re*(pattern: string, options = ""): Regex = initRegex(pattern, options)
+proc re*(pattern: string): Regex =
+  let (pattern, flags, study) = extractOptions(pattern)
```

Comment:
0.11.0 was the problematic release immediately followed by 0.11.2. It makes
no sense to support 0.11.0 but I'm mentioning it just in case.

On Fri, May 8, 2015 at 8:29 PM, Flaviu Tamas <notifications@github.com>
wrote:

> In src/nre.nim
> <https://github.com/flaviut/nre/pull/14#discussion_r29958982>:
>
> > @@ -427,7 +416,9 @@ proc initRegex(pattern: string, options: string): Regex =
> >
> >    result.captureNameToId = result.getNameToNumberTable()
> >
> > -proc re*(pattern: string, options = ""): Regex = initRegex(pattern, options)
> > +proc re*(pattern: string): Regex =
> > +  let (pattern, flags, study) = extractOptions(pattern)
>
> Sounds like I should set up the tester to test with 0.11.0, not just with
> devel. I'll look into it later today once I get to my PC.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/flaviut/nre/pull/14/files#r29958982>.
>